packages/block-editor/src/components/block-list/content.scss

```diff
@@ -161,13 +161,13 @@
 	padding: 0;
 }
 
-.is-content-locked {
-	.block-editor-block-list__block {
+.block-editor-block-list__layout,
+.block-editor-block-list__block {
+	pointer-events: initial;
+
+	&.is-editing-disabled {
 		pointer-events: none;
```

Comment:
We can't use the `Disabled` component / the `inert` HTML attribute because that completely disables all descendants of the node. We want _some_ children (e.g. the Post Title, Post Content) blocks to still be enabled.

Reply:
It might be nice to have some opacity at play here, e.g., lower opacity when `&.is-editing-disabled` and `full opacity for `:hover`. 

It might also be nice if this was a good idea.

Reply:
The problem with using `opactiy` (I tried it) is that it compounds. So e.g. if the opacity is 20% then a block that's 4 levels deep becomes `50% ^ 4 = 6.25%` opacity. I couldn't figure out a good solution to that.

Reply:
I think let's just start with this UI (the one we'll implement in https://github.com/WordPress/gutenberg/pull/49980) and iterate.